sklearn/feature_selection/sfs.py

```diff
@@ -0,0 +1,366 @@
+"""
+Sequential feature selection
+
+"""
+
+# Author: Sebastian Raschka <se.raschka@gmail.com>
+#
+# License: BSD 3 clause
+
+import numpy as np
+from itertools import combinations
+from collections import defaultdict
+from ..base import BaseEstimator
+from ..base import MetaEstimatorMixin
+from ..base import clone
+from ..utils.validation import check_is_fitted
+from ..externals import six
+from ..model_selection import cross_val_score
+from ..metrics import get_scorer
+
+
+class SFS(BaseEstimator, MetaEstimatorMixin):
```

Comment:
agreed! Tried that first though and was having some issues, i.e. `TypeError: Can't instantiate abstract class SFS with abstract methods _get_support_mask`. Do you know what would need to be added to make this work?